disnake/interactions/base.py

```diff
@@ -283,6 +285,27 @@ def followup(self) -> Webhook:
         }
         return Webhook.from_state(data=payload, state=self._state)
 
+    @utils.cached_slot_property("_cs_expires_at")
+    def expires_at(self) -> datetime:
+        """:class:`datetime.datetime`: Returns the interaction's expiration time in UTC.
+
+        This is exactly 15 minutes after the interaction was created.
+
+        .. versionadded:: 2.5
+        """
+        return self.created_at + timedelta(minutes=15)
+
+    def is_expired(self) -> bool:
+        """Whether the interaction is older than 15 minutes.
+
+        This does not take into account if the interaction was responded to within the 3 second limit.
```

Comment:
```suggestion
        This does not take into account if the interaction was already responded to.
```

Reply:
hm, I don't think that fully gets across what I'm trying to say here.

If the interaction was not responded to, but it has been 3 seconds, then the interaction is technically expired, although this method will return False.

Reply:
maybe something like `This does not take the 3 second limit for the initial response into account.` instead?

Reply:
https://github.com/DisnakeDev/disnake/pull/469/commits/4119a67432df50413249095163219c2401d69f16